sdk/maps/Azure.Maps.Search/src/MapsSearchClient.cs

```diff
@@ -1064,31 +370,39 @@ public virtual async Task<SearchAddressBatchOperation> SearchAddressBatchAsync(W
         }
 
         /// <summary>
-        /// The Search Address Batch API sends batches of queries to <see href="https://docs.microsoft.com/rest/api/maps/search/getsearchaddress">Search Address API</see> using just a single API call. You can call Search Address Batch API to run either asynchronously (async) or synchronously (sync). The async API allows caller to batch up to **10,000** queries and sync API up to **100** queries.
+        /// Translate a coordinate (example: 37.786505, -122.3862) into a human understandable street address. Most often this is needed in tracking applications where you receive a GPS feed from the device or asset and wish to know what address where the coordinate is located. This endpoint will return address information for a given coordinate.
         /// </summary>
-        /// <param name="waitUntil"> <see cref="WaitUntil.Completed"/> if the method should wait to return until the long-running operation has completed on the service; <see cref="WaitUntil.Started"/> if it should return immediately after starting the operation. </param>
-        /// <param name="queries"> The list of address geocoding queries/requests to process. The list can contain  a max of 10,000 queries and must contain at least 1 query. </param>
+        /// <param name="coordinates"> The coordinates of the location that you want to reverse geocode. Example: &amp;coordinates=lon,lat. </param>
+        /// <param name="resultTypes">
+        /// Specify entity types that you want in the response. Only the types you specify will be returned. If the point cannot be mapped to the entity types you specify, no location information is returned in the response.
+        /// Default value is all possible entities.
+        /// A comma separated list of entity types selected from the following options.
+        ///
+        /// <list type="bullet">
+        /// <item>Address</item>
+        /// <item>Neighborhood</item>
+        /// <item>PopulatedPlace</item>
+        /// <item>Postcode1</item>
+        /// <item>AdminDivision1</item>
+        /// <item>AdminDivision2</item>
+        /// <item>CountryRegion</item>
+        /// </list>
+        /// These entity types are ordered from the most specific entity to the least specific entity. When entities of more than one entity type are found, only the most specific entity is returned. For example, if you specify Address and AdminDistrict1 as entity types and entities were found for both types, only the Address entity information is returned in the response.
+        /// </param>
+        /// <param name="view">
```

Comment:
We should replace `view` to `localizedMapView` with type `LocalizedMapView` (Please refer to original code)

Reply:
@dubiety done